Doc/whatsnew/3.12.rst

```diff
@@ -226,6 +239,19 @@ although there is currently no date scheduled for their removal.
   :keyword:`for`, :keyword:`if`, :keyword:`in`, :keyword:`is` and :keyword:`or`.
   In a future release it will be changed to a syntax error. (:gh:`87999`)
 
+* :func:`asyncio.get_event_loop` emits a :exc:`DeprecationWarning` if there is
```

Comment:
The deprecation plans for this function have changed. See https://github.com/python/cpython/pull/100970 for the updated deprecation notice.

Reply:
Thanks for the note! This PR has got pretty out of date...